packages/frontend/src/screens.scss

```diff
@@ -0,0 +1,16 @@
+/* screens sizes
+
+320px — 480px: Mobile devices.
+481px — 768px: iPads, Tablets.
+769px — 1024px: Small screens, laptops.
+1025px — 1200px: Desktops, large screens.
+1201px and more — Extra large screens, TV.*/
+
+/* using with max-width */
+
+$screen-s: 480px; //orientation: portrait
+$screen-m: 768px; //oriention: portrait
+$screen-l: 1024px; //orientation: landscape
+$screen-xl: 1200px; //orientation: landscape
+
+$screen-xxl: 1201px; //using with min-width, orientation: landscape
```

Comment:
Not sure of the purpose of this file, because it is already defined in bootstrap